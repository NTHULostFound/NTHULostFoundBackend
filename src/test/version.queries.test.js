import { createTestClient } from 'apollo-server-testing'
import { gql } from 'apollo-server'
import graphqlServer from '../graphql'

const GET_VERSION = gql`
query Query {
    version
}
`

describe('[Queries.Version]', () => {
  it('Get version', async () => {
    const { query } = createTestClient(graphqlServer)

    const res = await query({
      query: GET_VERSION
    })

    expect(res).toMatchSnapshot()
  })
})
