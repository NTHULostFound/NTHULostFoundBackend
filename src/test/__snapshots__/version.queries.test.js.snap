// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Queries.Version] Get version 1`] = `
Object {
  "data": Object {
    "version": "1.0.0",
  },
  "errors": undefined,
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
